src/resources/views/base/inc/sidebar.blade.php

```diff
@@ -24,28 +24,27 @@
 @push('before_scripts')
   <script type="text/javascript">
     /* Recover sidebar state */
-    if (Boolean(sessionStorage.getItem('sidebar-collapsed'))) {
-      var body = document.getElementsByTagName('body')[0];
-      body.className = body.className.replace('sidebar-lg-show', '');
-    }
-
-    /* Store sidebar state */
-    var navbarToggler = document.getElementsByClassName("navbar-toggler");
-    for (var i = 0; i < navbarToggler.length; i++) {
-      navbarToggler[i].addEventListener('click', function(event) {
-        event.preventDefault();
-        if (Boolean(sessionStorage.getItem('sidebar-collapsed'))) {
-          sessionStorage.setItem('sidebar-collapsed', '');
-        } else {
-          sessionStorage.setItem('sidebar-collapsed', '1');
-        }
-      });
+    var body = document.getElementsByTagName('body')[0];
+    if (sessionStorage.getItem("sidebar-collapsed") !== null) {
+      var sidebarDefault = body.classList.contains("sidebar-lg-show");
+      if (Boolean(sessionStorage.getItem('sidebar-collapsed')) && sidebarDefault) {
+        body.classList.remove("sidebar-lg-show");
+      }
+      if (!Boolean(sessionStorage.getItem('sidebar-collapsed')) && !sidebarDefault) {
+        body.classList.add("sidebar-lg-show");
+      }
```

Comment:
Unfortunately, this does not work. 
In case the `.sidebar-lg-show` class is hidden by default (via config), the class is toggled since `!sessionStorage.getItem("sidebar-collapsed")` is `true` on a first visit.

Reply:
@martijnb92 that changes everything 🤯 glad you notice it 🙌
Know I understand why it was previously saving values as "0" and "1".

Reply:
I'll add another review with the fix 👌